diff -urN aaa/gnuplot-3.8j.0/configure.in gnuplot-3.8j.0/configure.in
--- aaa/gnuplot-3.8j.0/configure.in	2003-07-24 14:10:16.000000000 +0200
+++ gnuplot-3.8j.0/configure.in	2003-08-07 01:39:11.000000000 +0200
@@ -764,9 +764,6 @@
   cp ${srcdir}/tutorial/eg3.dat tutorial
 fi
 
-dnl build lisp files
-AC_CONFIG_SUBDIRS(lisp)
-
 dnl Substitute variables
 AC_SUBST(PACKAGE)
 AC_SUBST(PKG_MAJOR)
diff -urN aaa/gnuplot-3.8j.0/Makefile.am gnuplot-3.8j.0/Makefile.am
--- aaa/gnuplot-3.8j.0/Makefile.am	2003-01-30 19:01:05.000000000 +0100
+++ gnuplot-3.8j.0/Makefile.am	2003-08-07 01:42:19.000000000 +0200
@@ -1,7 +1,7 @@
 ## Process this file with automake to produce Makefile.in -*-Makefile-*-
 AUTOMAKE_OPTIONS = foreign 1.2h
 
-SUBDIRS = config m4 term src docs lisp man demo tutorial
+SUBDIRS = config m4 term src docs man demo tutorial
 
 EXTRA_DIST = BUGS CodeStyle Copyright FAQ GNUmakefile INSTALL INSTALL.gnu \
 Makefile.maint PATCHLEVEL PGPKEYS PORTING README README.1ST README.exp \
