--- gnuplot-4.4.3/configure.in.orig	2011-02-20 21:43:31.000000000 +0100
+++ gnuplot-4.4.3/configure.in	2011-03-16 15:09:34.731414618 +0100
@@ -680,7 +680,7 @@
   [test -z "${with_lua}" && with_lua=yes])
 
 if test "${with_lua}" = yes ; then
-  PKG_CHECK_MODULES(LUA, [lua], LUAFOUND=yes, [PKG_CHECK_MODULES(LUA, [lua5.1], LUAFOUND=yes, [LUAFOUND=no])])
+  PKG_CHECK_MODULES(LUA, [lua51], LUAFOUND=yes, [PKG_CHECK_MODULES(LUA, [lua5.1], LUAFOUND=yes, [LUAFOUND=no])])
   if test $pkg_failed != no; then
     AC_MSG_WARN([Could not find support for lua using pkg-config.])
     with_lua=no
