diff -Nru gnuplot-3.7.1/Makefile.in gnuplot-3.7.1.new/Makefile.in
--- gnuplot-3.7.1/Makefile.in	Sun Nov  7 15:01:04 1999
+++ gnuplot-3.7.1.new/Makefile.in	Sun Nov  7 15:01:41 1999
@@ -351,18 +351,18 @@
 ################################################################
 
 install: all $(LASERGNU)
-	$(srcdir)/mkinstalldirs $(bindir)
-	$(srcdir)/mkinstalldirs $(datadir)
-	$(srcdir)/mkinstalldirs $(mandir)/man1
-	$(INSTALL_PROGRAM) gnuplot $(bindir)/gnuplot
-	test ! -f gnuplot_x11 || $(INSTALL_PROGRAM) gnuplot_x11 $(bindir)/gnuplot_x11
-	-$(INSTALL_DATA) $(srcdir)/docs/gnuplot.1 $(mandir)/man1/gnuplot.1
+	$(srcdir)/mkinstalldirs $(DESTDIR)$(bindir)
+	$(srcdir)/mkinstalldirs $(DESTDIR)$(datadir)
+	$(srcdir)/mkinstalldirs $(DESTDIR)$(mandir)/man1
+	$(INSTALL_PROGRAM) gnuplot $(DESTDIR)$(bindir)/gnuplot
+	test ! -f gnuplot_x11 || $(INSTALL_PROGRAM) gnuplot_x11 $(DESTDIR)$(bindir)/gnuplot_x11
+	-$(INSTALL_DATA) $(srcdir)/docs/gnuplot.1 $(DESTDIR)$(mandir)/man1/gnuplot.1
 	-cd docs && $(MAKE) $(MFLAGS) install
 	@LINUXSUID@
 
 lasergnu_install: lasergnu docs/lasergnu.1
-	$(INSTALL_SCRIPT) $(srcdir)/lasergnu $(bindir)/lasergnu
-	$(INSTALL_DATA) $(srcdir)/docs/lasergnu.1 $(mandir)/man1/lasergnu.1
+	$(INSTALL_SCRIPT) $(srcdir)/lasergnu $(DESTDIR)$(bindir)/lasergnu
+	$(INSTALL_DATA) $(srcdir)/docs/lasergnu.1 $(DESTDIR)$(mandir)/man1/lasergnu.1
 
 lasergnu_noinstall:
 
diff -Nru gnuplot-3.7.1/docs/Makefile.in gnuplot-3.7.1.new/docs/Makefile.in
--- gnuplot-3.7.1/docs/Makefile.in	Sun Nov  7 15:01:04 1999
+++ gnuplot-3.7.1.new/docs/Makefile.in	Sun Nov  7 14:59:23 1999
@@ -349,15 +349,15 @@
 
 # For Unix and MSDOS only
 install: gnuplot.gih
-	$(top_srcdir)/mkinstalldirs $(datadir)
-	$(INSTALL_DATA) gnuplot.gih $(datadir)/gnuplot.gih
+	$(top_srcdir)/mkinstalldirs $(DESTDIR)$(datadir)
+	$(INSTALL_DATA) gnuplot.gih $(DESTDIR)$(datadir)/gnuplot.gih
 
 uninstall:
 	rm -f $(datadir)/gnuplot.gih
 
 # for VMS only
 install-vms: gnuplot.hlp
-	$(INSTALL_DATA) gnuplot.hlp $(datadir)/gnuplot.hlp
+	$(INSTALL_DATA) gnuplot.hlp $(DESTDIR)$(datadir)/gnuplot.hlp
 
 distdir = $(top_builddir)/$(PACKAGE)-$(VERSION)/$(subdir)
 
