--- gnuplot-3.7.2/configure.in.orig	Thu Jan 31 20:22:26 2002
+++ gnuplot-3.7.2/configure.in	Sun Aug  4 00:28:26 2002
@@ -390,12 +390,12 @@
 [#ifdef sun
 yes
 #endif
-], AC_CHECK_LIB(suntool, window_create,
-    AC_CHECK_HEADER(suntool/sunview.h,
-    AC_DEFINE(SUN, 1,
+], [AC_CHECK_LIB(suntool, window_create,
+    [AC_CHECK_HEADER(suntool/sunview.h,
+    [AC_DEFINE(SUN, 1,
               [ Define if you want to use the sunview terminal (sun). ])
-    TERMLIBS="-lsuntool -lsunwindow -lpixrect $TERMLIBS"),,
-    [-lsunwindow -lpixrect]))
+    TERMLIBS="-lsuntool -lsunwindow -lpixrect $TERMLIBS"])],,
+    [-lsunwindow -lpixrect])])
 
 dnl Process enable options
 
