--- gnuplot-3.7.1/m4/path-lib.m4.orig	Thu Aug 19 16:20:19 1999
+++ gnuplot-3.7.1/m4/path-lib.m4	Sat Oct 20 21:06:36 2001
@@ -14,11 +14,7 @@
 AC_TRY_LINK(dnl
 ifelse([$2], [main], , dnl Avoid conflicting decl of main.
 [/* Override any gcc2 internal prototype to avoid an error.  */
-]ifelse(AC_LANG, CPLUSPLUS, [#ifdef __cplusplus
-extern "C"
-#endif
-])dnl
-[/* We use char because int might match the return type of a gcc2
+ /* We use char because int might match the return type of a gcc2
     builtin and then its argument prototype would still apply.  */
 char $2();
 ]),
